### PR TITLE
[Player 3918] Fixed accessibility area blinking issue.

### PR DESCRIPTION
**Pull request by SZadorozhnyiEpam**
Fixed accessibility area blinking issue. Increased area of button for better user experience

### CODE REVIEW

**Comment by SZadorozhnyiEpam** [comment]
image
By "Increased area of button" I also mean that TalkBack highlight area now also changed (Green area). (Red area like it was before)

File: sdk/react/upNext.js [hunk 1]
```diff
@@ -123,19 +123,21 @@ class UpNext extends React.Component {
     if(this.isWithinShowUpNextBounds() && !this.props.upNextDismissed && upNextConfig.showUpNext === true && !this.props.ad && this.props.nextVideo != null) {
       var countdown = this.renderCountdownTimer();
       var upNextImage = (
+      <TouchableHighlight
+      style={styles.thumbnail}
+      accessible={true}
```
Comment (AlinaVoronkovaEpam) [inline sdk/react/upNext.js]: Please add spaces here according to the code style guide.

**Review by AlinaVoronkovaEpam — APPROVED** [review]
Fix code according to the code style guide

**Review by IvanSakharovskii — APPROVED** [review]

**Review by brol1dev — CHANGES_REQUESTED** [review]
Just attend @AlinaVoronkovaEpam comments. LGTM otherwise.

**Review by akshaydch11 — APPROVED** [review]

File: sdk/react/upNext.js (hunk 1 above)
Comment (SZadorozhnyiEpam) [reply sdk/react/upNext.js]: done

**Comment by AlinaVoronkovaEpam** [comment]
Looks good to me.

**Pull request merged**